### ISSUE TITLE
Sprint 2 Video Demo- Due 2/28

### ISSUE THREAD

**Issue by rhfallon** [Shrenil19/BrainyPhonics#23]
DUE 2/28 

In a 3-5 minute video, the team must present a demonstration of the functionality developed during that sprint.  The team should compare the features demo’d to the iteration plan in the Customer Agreement.  A copy of the demo video (or, if requested by the customer, a live demonstration), must be provided to the customer.  

In addition to the demonstration of the product’s functionality, the video can include other visual aids to help the audience understand how the features demoed relate to the iteration plan. 

In summary, we should:

1. Demo functionality completed during the sprint
2. Compare the functionality completed to what we set out for ourselves in the iteration plan
3. Use visual aids as needed to demonstrate our progress
4. Send a copy to Dr. Evans upon completion
5. 

_Demo should be moderately scripted to reduce filler words._ 

Full description can be found on [canvas](https://gatech.instructure.com/courses/169364/assignments/685612)
